doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -90,7 +90,7 @@ See `Matter samples`_ for the list of changes for the Matter samples.
 Matter fork
 +++++++++++
 
-The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from the upstream Matter repository up to, and including, ``bc6b43882a56ddb3e94d3e64956bd5f3292b4058``.
+The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from the upstream Matter repository up to, and including, ``1.0.0.2`` tag.
 
 The following list summarizes the most important changes inherited from the upstream Matter:
```

Comment:
Any changes worth of mentioning in the list in line 97?

Reply:
I think we could mention new log regions filtering and memory optimizations we introduced.

Reply:
Oh ok, my bad I thought those are not patches anymore, but parts of upmerge.

Reply:
OK, up to you what should be mentioned here, but can we add at least the information that the basic cluster was renamed? :)

Reply:
@kkasperczyk-no 1.0.0.2 contains most changes up to mid-December so not this time :)
@greg-fer Right, already added 3 points, but will also mention about Basic cluster.